Source/Fuse.Controls.Native/iOS/InputDispatch.uno

```diff
@@ -103,29 +103,31 @@ namespace Fuse.Controls.Native.iOS
 			return new RootInfo(rootVisual, rootView);
 		}
 
-		const int MaxPointerCount = 10;
-
-		static ObjC.Object[] _activePointers = new ObjC.Object[MaxPointerCount];
+		static List<ObjC.Object> _activePointers = new List<ObjC.Object>();
 
 		static int GetPointIndex(ObjC.Object uiTouch)
 		{
 			var firstUnused = -1;
-			for (var i = 0; i < MaxPointerCount; ++i)
+			for (var i = 0; i < _activePointers.Count; ++i)
 			{
 				var pointer = _activePointers[i];
 				if (CompareUITouch(pointer, null) && firstUnused < 0)
 					firstUnused = i;
 				else if (CompareUITouch(pointer, uiTouch))
 					return i;
 			}
-
+			if (firstUnused < 0)
+			{
+				_activePointers.Add(uiTouch);
+				firstUnused = _activePointers.IndexOf(uiTouch);
```

Comment:
This IndexOf is a bit pointless; you can just take `_activePonters.Count - 1` ;)